googlemock/README.md

```diff
@@ -53,18 +53,18 @@ the Apache License, which is different from Google Mock's license.
 If you are new to the project, we suggest that you read the user
 documentation in the following order:
 
-  * Learn the [basics](../../master/googletest/docs/primer.md) of
+  * Learn the [basics](../googletest/docs/primer.md) of
```

Comment:
It would be helpful when working with documentation in another branch, so links would refer to the current branch.

Also, the tool that I used to check links doesn't support this links https://github.com/remarkjs/remark-validate-links

I can revert removing "master" if you want.